app/controllers/concerns/goodcity_sync.rb

```diff
@@ -0,0 +1,23 @@
+module GoodcitySync
+
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :set_request_from_stockit, only: [:update, :destroy, :create]
```

Comment:
@swatijadhav we can't really infer from these actions that the request actually comes in from Stockit. What if we start editting packages through Ember next week/month? However, once we do the authentication ticket, we will know that the request has come in from an API user, so no need to change this until that happens.